nailgun/entities.py

```diff
@@ -932,10 +967,14 @@ def read(self, entity=None, attrs=None, ignore=('content_view_filter',)):
             )
         if attrs is None:
             attrs = self.read_json()
-        # Field should be present in entity only if it was passed in attributes
-        for entity_field in entity.get_fields().keys():
-            if entity_field not in attrs:
-                del entity._fields[entity_field]
+        ignore = set(ignore).union([
```

Comment:
The handling of the `ignore` argument is another things I don't like about NailGun. See #134.